test/03typemap.t

```diff
@@ -8,6 +8,7 @@ BEGIN {
 use Test::More;
 use TestInlineSetup;
 use Inline Config => DIRECTORY => $TestInlineSetup::DIR;
+use File::Spec;
```

Comment:
This is already loaded by `TestInlineSetup`.

Reply:
Not anymore. The very same patch removes loading File::Spec from TestInlineSetup. Hence it  needs to be loaded somewhere else. Preferably where it used from. That is in 03typemap.t:14. The principle of locality. I personally don't like implicitly loaded dependencies because they easily break when the underlying module changes. If you don't agree, I can shuffle loading File::Spec back. I only thought that my approach made more sense.

Reply:
I am asking you to leave it as it was. Sorry if that wasn't clear. If you wanted to make such changes, it should be done in a separate commit (which I would, unless there were a pressing reason, not accept).

Reply:
Fixed in 003efa7f1855340fd6b4317ac7f1dee7bb895040.